.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd "$SRC"
+
+declare -a PROJECTS=(amt hamt)
+declare -A PROJECT_PATHS=(
+	[amt]="ref-fvm/ipld/amt/fuzz"
+	[hamt]="ref-fvm/ipld/hamt/fuzz"
+)
+
+export CARGO_TARGET_DIR="$SRC/target"
+FUZZ_TARGET_OUTPUT_DIR="$CARGO_TARGET_DIR/x86_64-unknown-linux-gnu/release"
+
+
+for project in "${PROJECTS[@]}"; do
+	pushd "${PROJECT_PATHS[$project]}"
+
+	cargo +nightly fuzz build -O --debug-assertions
```

Comment:
Shouldn't we be using the same version as the tool chain?

Reply:
Unfortunately, we can't. `cargo fuzz` uses some options which are only available in unstable builds. I think it will change once 1.59.0 is adopted by cargo-fuzz but I might be wrong (and we are on 1.58 either way).